### ISSUE TITLE
SSH does not try IPv4 after IPv6 fails (when DNS has addresses for both)

### ISSUE THREAD

**Issue by box293** [FreeRDP/Remmina#953]
```
Linux Mint 17.3 Rosa MATE
Remmina version 1.2.0-rcgit.14
Desktop environment : MATE
Connecting to (OS and version): any linux SSH session
Connecting via (RDP, VNC, ...): SSH
```

I've noticed an unusual behaviour which I can reproduce.

If Remmina cannot connect to the DNS destination using IPv6, it doesn't try IPv4.

I've been setting up some servers to do some application testing. 
I already have entries in my DNS server for IPv4 and IPv6 for the new server.
```
core-030.box293.local has address 10.25.5.160
core-030.box293.local has IPv6 address 2001:44b8:3132:25:10:25:5:160
```
After deploying the servers from a template, they initially have their IPv4 address defined and active but not the IPv6 address.
If I try and connect to the dns name **core-030** it fails with the error:
```
Failed to startup SSH session: Timeout connecting to to core-030
```
If I jump to my Linux Mint terminal session and do a:
```
ssh root@core-030
```
It takes a little longer than usual but then eventually connects. I'm assuming at this point it tries and connects to the IPv6 address, fails, and then trys the IPv4 address and succeeds.

My assumption is that Remmina tries to connect to the IPv6 address and fails. It doesn't then try the  IPv4 address.

I'm not sure how all of this happens in the back end. I assume that the Linux Mint terminal resolves the core-030 object, receives an IPv6 and IPv4 record and tries the IPv6 record first (perhaps it tries the address based on the IP binding order).

**Comment by antenore** [comment]
True, there is not a fallback mechanism at all in Remmina.

I'll investigate furthermore but I suspect that is a client name resolution issue on your mint.

**Comment by antenore** [comment]
Remmina uses libssh, take a look at this thread:

https://red.libssh.org/issues/101 

Which version of libssh do you have, I cannot find at http://distrowatch.com/table.php?distribution=mint&pkglist=true&version=17.3#pkglist (there is only libssh2)

I'm checking if it's solved in libssh 0.7 but I doubt.

**Comment by box293** [comment]
Yep that seems to be the issue I'm having.
```
dpkg -l libssh*
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                    Version          Architecture     Description
+++-=======================-================-================-===================================================
ii  libssh-4:amd64          0.6.1-0ubuntu3.3 amd64            tiny C SSH library
ii  libssh2-1:amd64         1.4.3-2          amd64            SSH2 client-side library
```

It's not a major issue, once I activate the IPv6 address it works fine because it can connect. Just need to wait for it to be resolved in libssh.

Thanks for looking.
